pkg/operator/ceph/cluster/mon/mon_test.go

```diff
@@ -123,6 +123,19 @@ func newCluster(context *clusterd.Context, namespace string, hostNetwork bool, a
 	}
 }
 
+// testStart will orchestrate the mons with the existing settings on the mon cluster
+func testStart(c *Cluster) (*cephconfig.ClusterInfo, error) {
+	return c.Start(c.clusterInfo, c.rookVersion, c.cephVersion, cephv1.MonSpec{Count: c.Count, AllowMultiplePerNode: c.AllowMultiplePerNode},
+		rookalpha.Placement{}, v1.ResourceRequirements{})
+}
+
+func setTestMonSettings(c *Cluster, currentMons int, mon cephv1.MonSpec, rookVersion string) {
```

Comment:
I'm not super clear on what the purpose of this is. Maybe a comment or rename would help?

Reply:
this could be simplified by using a cluster spec as well, but if i keep it i'll at least add a comment